actioncable/test/test_helper.rb

```diff
@@ -41,3 +41,5 @@ def wait_for_executor(executor)
     end
   end
 end
+
+require_relative "../../tools/junit_format_test_report_for_buildkite" if ENV["BUILDKITE"]
```

Comment:
I'm curious. You use `BUILDKITE_JOB_ID` in a plugin, but you use `BUILDKITE` here. Is there a case where you want to generate a report even if `BUILDKITE_JOB_ID` is missing? 
Unless there is a specific reason, I think that the environment variables to be referenced should be unified.

Reply:
IMO, `BUILDKITE` env is suitable to check that current test environment is buildkite or not.
`BUILDKITE_JOB_ID` is used to calculate store location at this time but it depends on buildkite architecture. Even if we need to change the location like `ENV['BUILDKITE_ARTIFACT_LOCATION']`, we don't have to change this condition.

Reply:
@y-yagi What do you think?

Reply:
:+1: